p2p/protocol/identify/id.go

```diff
@@ -368,7 +368,7 @@ func (ids *IDService) identifyConn(c network.Conn, signal chan struct{}) {
 
 	// ok give the response to our handler.
 	if err = msmux.SelectProtoOrFail(ID, s); err != nil {
-		log.Event(context.TODO(), "IdentifyOpenFailed", c.RemotePeer(), logging.Metadata{"error": err})
+		// log.Event(context.TODO(), "IdentifyOpenFailed", c.RemotePeer(), logging.Metadata{"error": err})
```

Comment:
Is there a metric that should be emitted here in leiu of depreciated go-log Events?

Reply:
Just a structured log message.